test.js

```diff
@@ -93,3 +93,13 @@ test('all preferred ports in array are unavailable', async t => {
 	t.not(port, desiredPorts[0]);
 	t.not(port, desiredPorts[1]);
 });
+
+test('makeRange', t => {
```

Comment:
Shouldn't the tests be more descriptive? It's not clear what's the purpose of each test we're performing, i.e., what scenarios are we verifying?

Reply:
done